packages/jaeger-ui/src/components/TracePage/TraceStatistics/tableValuesTestTrace/traceWithTwoNonOverlappingChildren.json

```diff
@@ -32,7 +32,7 @@
           "spanID": "006c3cf93508f205"
```

Comment:
Similar to timestamps, these test files should really be using manually defined span IDs, like `parentSpan`, `childSpanA`, ...

If I were writing these tests, I would've created the spans directly in the code, with required values.

Reply:
> I would've created the spans directly in the code, with required values.

How will you do this?? is there any function to create the span directly in code?

Reply:
There may be functions somewhere, but the span structure is pretty simple, it's easy to define a local helper function that takes only the arguments that need to vary across the tests, like span id, parent id, start / end, maybe operation name if group-by is being tested